docs/dev/check_references.md

```diff
@@ -99,6 +111,76 @@ The complete list of mandatory and optional attributes for the `manifest.json` f
+| `author[name]`                                       | String                                                                  | Mandatory                                  | The human-readable name for this company.                                                                                                                                                                                                                  |   |
+| `author[sales_email]`                                | String (Email)                                                          | Mandatory                                  | The email to contact for any subscription-level events.                                                                                                                                                                                                    |   |
+| `author[support_email]`                              | String (Email)                                                          | Mandatory                                  | The email to contact for any support and maintenance queries.                                                                                                                                                                                                  |   |
+| `author[vendor_id]`                                  | String                                                                  | Mandatory                                  | The vendor ID to use for subscription purposes. Must be globally unique and cannot be changed. Should follow the strict standards of `integration_id` where only dashes and alphabetic chars are allowed. This value is provided to partners. |   |
+| `classifier_tags`                                    | Array of String                                                         | Mandatory, can be `[ ]`                    | Some classifier information about this app. This includes information such as `supported_os` and `available_offerings`.                                                                                                                            |   |
+| `display_on_public_website `                         | Boolean                                                                 | Mandatory                 | Whether or not information about this listing is displayed on the public Datadog docs site. Once this is set to True, it cannot be changed.                                                                                                               |   |
```

Comment:
We should confirm with someone on the docs team if the "cannot be changed" constraint is still valid
@davidejones would you be able to confirm if this is still a constraint?